src/javascript/app/pages/user/account/settings/professional_client.js

```diff
@@ -53,9 +53,13 @@ const professionalClient = (() => {
                 $(popup_selector).find('#btn_accept, #btn_decline').off('click').on('click dblclick', function() {
                     if ($(this).attr('data-value') === 'decline') {
                         $chk_professional.prop('checked', false);
+                        $('#trading_experience_form').setVisibility(1);
                     }
                     $('#professional_popup').remove();
                 });
+                $('#trading_experience_form').setVisibility(0);
+            } else {
+                $('#trading_experience_form').setVisibility(1);
```

Comment:
if we're going to call this element many times, better to store it outside, e.g. on line 41 etc, something like `const $trading_experience_form = $('#trading_experience_form');`